st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Couldn't we do this as part of the regular app bootstrap process - the same way we ensure MongoDB indexes on startup?